### ISSUE TITLE
Improve individual and sample data models

### ISSUE THREAD

**Issue by pfurio** [opencb/opencga#701]
In order to be consistent in the way things have been done so far, we would need to apply some changes to some of the data models as well as to some webservices API.

Because Individuals contains Samples, the Individual data model should contain a list of samples. This was recently done. However, Sample data model still contain an Individual in the data model. This Individual should be marked to deprecated and, in the future, should be completely gone.

On the other hand, although Individual already contains a list of samples, every time the user queries an Individual, the list of samples belonging to that Individual is not being returned. However, this list should be also provided.

Due to these changes, it would necessary to do some changes in the webservices API as well. At the moment, _create_ and _update_ Sample webservices can assign a Sample to an Individual. However, though we do not plan to remove these features, we think we should be able to pass a list of samples in the _create_ and _update_ Individual webservices and do it the other way round as well.

To sum up, the list of actions would be:
- [x] Deprecate Individual from Sample data model.
- [x] Return the list of samples when fetching Individual information.
- [x] Add list of samples to _create_ Individual webservice
- [ ] Add list of samples to _update_ Individual webservice
